sdk/core/core-amqp/src/index.ts

```diff
@@ -28,8 +28,8 @@ export {
   Dictionary,
   Message as AmqpMessage,
   isAmqpError,
-  MessageHeader as AmqpMessageHeader,
-  MessageProperties as AmqpMessageProperties
+  MessageHeader as RheaAmqpMessageHeader,
+  MessageProperties as RheaAmqpMessageProperties
```

Comment:
Same here.
Though this changes the API Surface.. event-hubs is not affected since it doesn't use these interfaces.

Only Service Bus V7 is affected